bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
+	var fileBytes []byte
+	var err error
+
+	lowerPath := strings.ToLower(path)
+
+	if strings.Contains(lowerPath, "http") {
+		resp, err := http.Get(lowerPath)
+		if err != nil {
+			return fmt.Errorf("Could not get remote file")
+		}
+		defer resp.Body.Close()
+
+		if resp.StatusCode >= 400 {
+			return fmt.Errorf("Invalid status code %d loading remote file", resp.StatusCode)
```

Comment:
Can we parse the URI and display the URI in message without the potential `username-password`?

Reply:
do we know how the username/password will be keyed off of - we weren't sure if this would be consistent across all users (pretty hard to guarantee).

Reply:
Yes, it is standard as outlined in the ADR. My guess is the URI parser will separate it out for you so you can get at the URL part to log.